frontend/src/lib/components/Pagination.svelte

```diff
@@ -8,7 +8,7 @@
 	export let pagination: Pagination;
 	export let prevPage: string | undefined = undefined;
 	export let nextPage: string | undefined = undefined;
-	export let element: HTMLElement | undefined = undefined;
+	export let element: HTMLElement;
```

Comment:
Instead of passing in an element into pagination, dispatch a nextPageClicked and prevPageClicked event.  Then listen to those events in the +page.svelte component and call the element.scrollIntoView(); function there.  

Otherwise you aren't following separation of concerns and are forcing the pagination component to directly manage the scroll behavior of an element when that isn't something that it has knowledge of.

Reply:
To see this bug, go to the Table view, open the dev console, and click the Pagination icons in the Table View

```
TypeError: Cannot read properties of undefined (reading 'scrollIntoView')
    at HTMLAnchorElement.scrollToTop (Pagination.svelte:14:11)
```